crates/testing/tests/tests_1/quorum_proposal_recv_task.rs

```diff
@@ -210,7 +210,10 @@ async fn test_quorum_proposal_recv_task_liveness_check() {
                 ),
             ),
         )),
-        quorum_proposal_missing(),
+        exact(QuorumProposalRequestSend(
+            ViewNumber::new(2),
+            handle.public_key()
+        )),
```

Comment:
Not as familiar with the integration tests - why do we expect a request for view 2 here?

Reply:
That proposal is missing in the parent's state. Looking here
```rust
    let inputs = vec![serial![QuorumProposalRecv(
        proposals[2].clone(),
        leaders[2]
    )]];
```

We were sent a proposal for view 3, but the proposal for view 2 (i.e. the `justify_qc`) does *not* exist in memory due to the setup of the test, so we broadcast a request for the proposal from the leader to get the proposal so we know that we can safely branch off of this node.

Reply:
Awesome, that's what I suspected, thanks!